iamctl/cmd/cli/exportAll.go

```diff
@@ -41,11 +41,12 @@ var exportAllCmd = &cobra.Command{
 		if outputDirPath == "" {
 			outputDirPath = baseDir
 		}
-
```

Comment:
Let's keep the empty line and for consistency let's stick to the same order as in import. Export claims first.